postgresql/default.toml

```diff
@@ -11,6 +12,8 @@ password = 'admin'
 [replication]
 name = 'replication'
 password = 'replication'
+synchronous = true
+synchronous_standby_names = '*'
```

Comment:
Don't know if this should be editable. Ideally hab would just figure everything out by itself.